test/FingerprintApiTest.php

```diff
@@ -203,4 +210,17 @@ public function testGetVisitsWithLimit()
         $count = count($visits->getVisits());
         $this->assertLessThanOrEqual($limit, $count);
     }
+
+    public function testGetEventRawResponse()
```

Comment:
Please add more tests. For example, pass the wrong type to the field, add new fields. First one will break the API, so it should at least provide ability to get raw data.

Reply:
added serialization and rawResponse test cases

Reply:
I mean you need to really break it. Take correct mock and replace string fields with an object, or remove required field.

Reply:
This is already done in another test called `testGetBrokenFormatEvent`